tests/test_ignore_order.py

```diff
@@ -779,3 +779,166 @@ def test_ignore_order_and_group_by(self):
         diff2 = DeepDiff(t1, t2, group_by='id', ignore_order=True)
         expected2 = {'iterable_item_added': {"root['BB']['ate'][1]": 'Brownies'}}
         assert expected2 == diff2
+
+    def test_compare_func(self):
+        t1 = {
```

Comment:
Since the test objects are not small, could you please move them as fixtures to the `tests/fixtures` directory?